mscore/musescore.cpp

```diff
@@ -355,18 +355,16 @@ void MuseScore::closeEvent(QCloseEvent* ev)
       unloadPlugins();
       QList<MasterScore*> removeList;
       for (MasterScore* score : scoreList) {
-            if (score->created() && !score->dirty())
```

Comment:
Wouldn't it be easier or more straight forward to mark a score dirty on crteation?

Reply:
Perhaps, but it could have unintended conseqeuences. I wanted to make this fix as straightforward and localized as possible to minimize the chance of regressions.

In any case, both the “created” and “dirty” flags are needed anyway; for example, the “created” flag is how MuseScore knows that a file has never been saved and thus needs to have a filename assigned to it when saving. Since we're going to have two flags no matter what, it seems useful — or at the very least, potentially useful — to keep them logically distinct.

Reply:
OK, 't was just a random thought of mine ;-)